screwdriver.yaml

```diff
@@ -28,6 +28,8 @@ shared:
         requires:
           - ~pr
           - ~commit
+          - ~pr:vis-v2
+          - ~commit:vis-v2
```

Comment:
Just for pr builds?
```suggestion
```
Could add webservice pr build too

Reply:
:+1: If we need/want this we can add it later